cli/src/program.rs

```diff
@@ -246,6 +248,26 @@ impl ProgramSubCommands for App<'_, '_> {
                                      holds a large balance of SOL",
                                 ),
                         )
+                        .arg(
+                            Arg::with_name("max_sign_attempts")
+                                .long("max-sign-attempts")
+                                .takes_value(true)
+                                .validator(is_parsable::<u64>)
+                                .default_value("5")
+                                .help(
+                                    "Maximum number of attempts to sign or resign transactions \
+                                    after blockhash expiration.
+                                    If any transactions sent during the program deploy are still \
+                                    unconfirmed after the initially chosen recent blockhash \
+                                    expires, those transactions will be resigned with a new \
+                                    recent blockhash and resent. Use this setting to adjust \
+                                    the maximum number of transaction signing iterations. Each \
+                                    blockhash is valid for about 60 seconds, which means using \
+                                    the default value of 5 will lead to sending transactions \
+                                    for at least 5 minutes or until all transactions are confirmed,\
+                                    whichever comes first.",
+                                ),
```

Comment:
The line break is a little funky

```
        --max-sign-attempts <max_sign_attempts>
            Maximum number of attempts to sign or resign transactions after blockhash expiration.
                                                If any transactions sent during the program deploy are still unconfirmed
            after the initially chosen recent blockhash expires, those transactions will be resigned with a new recent
            blockhash and resent. Use this setting to adjust the maximum number of transaction signing iterations. Each
            blockhash is valid for about 60 seconds, which means using the default value of 5 will lead to sending
            transactions for at least 5 minutes or until all transactions are confirmed,whichever comes first. [default:
            5]
```

Reply:
Ah oops, sorry about that. Thanks for noticing -- fixed!